src/main/webapp/app/filters/cantaloupeUrlFilter.js

```diff
@@ -0,0 +1,8 @@
+sage.filter('cantaloupeUrl', function () {
+  return function (string) {
+    if (string.indexOf(appConfig.fedoraPath) !== -1) {
+        string = string.split(appConfig.fedoraPath,2)[1];
+    }
+    return appConfig.cantaloupeBaseUrl+btoa("fedora:"+string)+"/info.json";
```

Comment:
Cantaloupe resolver delegate is using UUIDs now

Reply:
can this upgrade be handled in an additional feature request?

Reply:
The resolver still supports the old behavior and I think we would need to discuss how the UUID approach could apply to Sage and Cap before trying to implement.